developer/src/kmc-keyboard/test/fixtures/sections/disp/typical.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE keyboard SYSTEM "../../../../../../../resources/standards-data/ldml-keyboards/techpreview/dtd/ldmlKeyboard.dtd">
+<keyboard locale="mt" conformsTo="techpreview">
+  <names>
+    <name value="disp-typical" />
+  </names>
+
+  <displays>
+    <display to="e" display="(e)"/>
+  </displays>
```

Comment:
I think we need one more test file with just `displayOptions` and no `display` entries

Reply:
done